src/test/java/org/jboss/windup/rules/tests/WindupRulesTest.java

```diff
@@ -319,6 +319,7 @@ private void runWindup(GraphContext context, File baseRuleDirectory, final List<
         windupConfiguration.setOutputDirectory(output.toPath());
         windupConfiguration.addDefaultUserRulesDirectory(baseRuleDirectory.toPath());
         windupConfiguration.setOptionValue(SourceModeOption.NAME, sourceMode);
+        windupConfiguration.setOffline(true);
```

Comment:
Unrelated change, but this keeps windup's xml validation from triggering on our test data. That is really unnecessary and sometimes slow.

Obviously, the build-time validation against the rules schema still happens, and that is the important part.